jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunction.kt

```diff
@@ -60,6 +60,7 @@ class CreateLambdaFunction(
 
         val element: PsiElement? = elementPointer?.element
         if (handlerName == null || element == null || lambdaHandlerResolver == null) {
+            e.presentation.isVisible = true
```

Comment:
if handlerName, resolver, element etc is null - why would we want to show the menu ?

Reply:
We show a form and make it filled in by the user. It works in 2018.3 without this line